exaudfclient/base/WORKSPACE

```diff
@@ -9,6 +9,36 @@ http_archive(
     url = "https://github.com/bazelbuild/bazel-skylib/releases/download/{}/bazel-skylib.{}.tar.gz".format(skylib_version, skylib_version),
 )
 
+RULES_JVM_EXTERNAL_TAG = "4.2"
```

Comment:
maybe we should update, or we do this in a extra PR

https://github.com/bazelbuild/rules_jvm_external/releases/tag/6.2

Reply:
Tried it, but it does not work. 
Anyway, will be updated in https://github.com/exasol/script-languages/pull/439